OpenSim/Common/Reporter.h

```diff
@@ -163,10 +163,24 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
     TableReporter_() = default;
     virtual ~TableReporter_() = default;
 
+    /** Retrieve the report.                                                  */
     const TimeSeriesTable_<ValueT>& getReport() const {
         return _outputTable;
     }
 
+    /** Clear the report. This can be used for example in loops where iteration
+    is intended to create a fresh report.                                     */
```

Comment:
I have added a sentence or two more. 
I don't think this is the place to explain details about component connections. Documentation on connections (or connectors) should do that.